### ISSUE TITLE
Large numbers should be written with underscores

### ISSUE THREAD

**Issue by rawsyntax** [rrrene/credo#149]
I ran into this problem when setting timeout configuration in HTTPoison.

This check has a problem when multiple large numbers are on the same line:

```elixir
defmodule TestProject do
  def method do
    [connect_timeout: 10_000, recv_timeout: 10_000, timeout: 10_000]
  end
end
```

Generates two warnings:
```sh
┃ [R] ↗ Large numbers should be written with underscores: 10_000
┃       lib/test_project.ex:3:60 (TestProject.method)
┃ [R] ↗ Large numbers should be written with underscores: 10_000
┃       lib/test_project.ex:3:44 (TestProject.method)
```

Whereas:

```elixir
defmodule TestProject do
  def method do
    [connect_timeout: 10_000]
  end
end
```

Does not generate a warning

**Comment by rrrene** [comment]
Hi, what's your Elixir, Erlang and Credo version?

**Comment by rawsyntax** [comment]
```
Erlang/OTP 18 [erts-7.3] [source] [64-bit] [smp:8:8] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]

Elixir 1.3.0
```

credo is 0.4.8

**Comment by rrrene** [comment]
The problem should go away with Elixir 1.3.2 - do you have the option to upgrade?

**Comment by rawsyntax** [comment]
Yes that's good enough for me
